src/engine/strat_engine/pool.rs

```diff
@@ -135,7 +141,8 @@ impl StratPool {
 
     /// Minimum initial size for a pool.
     pub fn min_initial_size() -> Sectors {
-        INITIAL_META_SIZE + INITIAL_DATA_SIZE + INITIAL_MDV_SIZE
+        // One extra meta for spare
+        (INITIAL_META_SIZE * 2u64) + INITIAL_DATA_SIZE + INITIAL_MDV_SIZE
```

Comment:
Yeah, it wouldn't be. I guess you do have to specify it to be _some_ unsigned type, and it won't infer one of the many acceptable types. I wonder why...